%YAML:1.0
---
model_type: PINHOLE
camera_name: camera
image_width: 1920
image_height: 1200
distortion_parameters:
   k1: -1.5855983900634696e-01
   k2: 1.2994555880814793e-01
   p1: -6.0424265983630317e-04
   p2: 9.1268093157433972e-04
projection_parameters:
   fx: 978.536621
   fy: 957.115245
   cx: 1009.157043
   cy: 614.557359
   tx: 0.000000000000e+00
   ty: 0.000000000000e+00
